imageio/imageio-tiff/src/main/java/com/twelvemonkeys/imageio/plugins/tiff/CCITTFaxDecoderStream.java

```diff
@@ -366,17 +382,7 @@ private int decodeRun(final Tree tree) throws IOException {
     }
 
     private void resetBuffer() throws IOException {
-        for (int i = 0; i < decodedRow.length; i++) {
-            decodedRow[i] = 0;
-        }
-
-        while (true) {
-            if (bufferPos == -1) {
-                return;
-            }
-
-            readBit();
-        }
+        bufferPos = -1;
```

Comment:
This change looks a bit suspicious... 
Though it could just as well be that the previous code wasn't needed. Which leads me to... Do you have a unit test? :-)